iXBRLViewerPlugin/viewer/src/html/inspector.html

```diff
@@ -177,8 +177,7 @@ <h3 class="collapsible-header" data-i18n="inspector.associatedFacts">Associated
 
     </div>
     <div class="inspector-foot powered-by">
-        <span data-i18n="inspector.poweredBy">Powered by</span>
-        <a href="https://www.workiva.co.uk/solutions/xbrl-and-ixbrl" target="_blank"><img alt="Workiva logo (opens in new window)" src="../img/workiva.svg" /></a>
+      mms
```

Comment:
Do we need this `<div class="inspector-foot powered-by" />` tag? I would remove it if possible.